docs/devsguide/new_dev_guide.rst

```diff
@@ -0,0 +1,106 @@
+*********************
+New Developer's Guide
+*********************
+
+=================
+Setting up Github
+=================
+
+
```

Comment:
Can you get rid of all of the whitespace between the headings and the text please?  This applies to the whole file.  Thanks!